DemoApp/LoginDemo/Utility/UIImageView+imageFromServerURL.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  UIImageView+imageFromServerURL.swift
+//  LoginDemo
+//
+//  Created by Marcin Religa on 30/6/22.
+//  Copyright © 2022 Intelligent Apps GmbH. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension UIImageView {
+    func imageFromServerURL(_ URLString: String, placeHolder: UIImage?) {
+        self.image = nil
```

Comment:
Shouldn't the placeholder go here?

Reply:
It depends if we want to always show the placeholder and then change it to a loaded image, or if we want to show placeholder only if loading the image fails.

I'd say for this demo it probably doesn't matter much.